packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -158,6 +157,23 @@ const verificationScreens = (Navigator: typeof Stack) => {
   )
 }
 
+const pincodeSetScreenOptions = ({
```

Comment:
This is the only part I don't like of new navigation - definitely feels like this should live on the component. What do you think about making this part of the screen's file and exporting it as opposed to doing it here?

Reply:
Yes I agree, we could easily adapt the old way and use `MyComponent.navigationOptions` inside `Navigator.tsx`.
We can discuss quickly with the rest of the team and do a global pass on this later on.